vivisect/tools/graphutil.py

```diff
@@ -629,7 +629,16 @@ def clearGraphRouting(graph):
     for nid, ninfo in graph.getNodes():
         graph.getNodeProps(nid)['up'  ] = False
         graph.getNodeProps(nid)['down'] = False
```

Comment:
(as an aside) direct manipulation of the node's property dict is going to be *very* wrong in the new graph API